pkg/health/const.go

```diff
@@ -27,19 +27,26 @@ const (
 )
 
 const (
-	IPAMSvc              string = "IPAM"
-	IPAMCliSvc           string = "IPAMCli"
-	NSPCliSvc            string = "NSPCli"
-	EgressSvc            string = "Egress"
-	NSMEndpointSvc       string = "NSMEndpoint"
-	TargetRegistryCliSvc string = "TargetRegistryCli"
-	StreamSvc            string = "Stream"
-	FlowSvc              string = "Flow"
-	NSPSvc               string = "NSP"
+	IPAMSvc        string = "IPAM"
+	IPAMCliSvc     string = "IPAMCli"
+	NSPCliSvc      string = "NSPCli"
+	EgressSvc      string = "Egress"
+	NSMEndpointSvc string = "NSMEndpoint"
+	StreamSvc      string = "Stream"
+	FlowSvc        string = "Flow"
+	NSPSvc         string = "NSP"
+	AmbassadorSvc  string = "Ambassador"
 )
 
 var LBReadinessServices []string = []string{NSPCliSvc, NSMEndpointSvc, EgressSvc, StreamSvc, FlowSvc}
-var FEReadinessServices []string = []string{NSPCliSvc, TargetRegistryCliSvc, EgressSvc}
+var LBLivenessServices []string = []string{NSMEndpointSvc}
+var FEReadinessServices []string = []string{NSPCliSvc, EgressSvc}
 var ProxyReadinessServices []string = []string{IPAMCliSvc, NSPCliSvc, NSMEndpointSvc, EgressSvc}
-var IPAMReadinessServices []string = []string{NSPCliSvc, IPAMSvc}
-var NSPReadinessServices []string = []string{NSPSvc}
+var ProxyLivenessServices []string = []string{NSMEndpointSvc}
+var IPAMReadinessServices []string = []string{NSPCliSvc}
+var IPAMLivenessServices []string = []string{IPAMSvc}
+var NSPReadinessServices []string = []string{}
```

Comment:
Why keeping the empty sub-services? (NSPReadinessServices, TAPAStartupServices and TAPAReadinessServices)

Reply:
Hmm, could be that these are just confusing, so I will remove them.